internal_testsuites/starlark/packages-with-replace/replace-with-local/main.star

```diff
@@ -0,0 +1,28 @@
+dependency = import_module("github.com/kurtosis-tech/sample-dependency-package/main.star")
+
+EXPECTED_MSG_FROM_MAIN = "dependency-loaded-from-main"
+EXPECTED_MSG_FROM_LOCAL_PACKAGE_MAIN = "msg-loaded-from-local-dependency"
+
+MSG_ORIGIN_MAIN = "main"
+MSG_ORIGIN_LOCAL_DEPENDENCY = "local"
+
+# TODO remove https://github.com/kurtosis-tech/sample-startosis-load/tree/main/sample-package if it's not used
```

Comment:
why is this comment here?

Reply:
it was a comment for me but I forgot to remove it, good catch!